src/teuthology_api/main.py

```diff
@@ -9,7 +9,7 @@
 
 load_dotenv()
 
-DEPLOYMENT = os.getenv("DEPLOYMENT")
+DEPLOYMENT = os.getenv("DEPLOYMENT", "development")
```

Comment:
I'm not sure if the default should be "development". If the variable is accidentally missing from prod deployment, then app will default to dev environment. On local dev environment, having the developer realise the auth is failing because of prod env's github requirement and then add `DEPOYMENT="development"`is easier and doesn't have any major bad consequences. There's already a note in README which encourages developers to set that variable on local development setup, so hopefully this should not cause problems most of the time.
```suggestion
DEPLOYMENT = os.getenv("DEPLOYMENT")
```